src/sql/setup.js

```diff
@@ -22,4 +22,7 @@ define(["require", "exports"], function (require) {
 	require.__$__nodeRequire('zone.js/dist/zone-error');
 	require.__$__nodeRequire('chart.js');
 	window["Zone"]["__zone_symbol__ignoreConsoleErrorUncaughtError"] = true;
+	window["Zone"]["__zone_symbol__unhandledPromiseRejectionHandler"] = e => setImmediate(() => {
```

Comment:
Out of curiosity - where are finding these values such as `__zone_symbol__unhandledPromiseRejectionHandler`? I did a quick search and the only things coming up are PRs like this one 😄 Are you just looking through the source or something?

Reply:
Looking through the source code of the library